<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>1068</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Классификатор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1321</width>
      <height>1011</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Обучение моделей</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Путь к файлу с данными:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="path_to_data">
         <property name="text">
          <string>C:\_Programming\BloodClassifierClassic\Data\cells_all.csv</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Способ классификации:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="classifier_type"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="preprocess_type">
         <item>
          <property name="text">
           <string>Стандартизация</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нормализация</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Способ предобработки данных:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>326</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_select_input_data">
         <property name="text">
          <string>Выбрать файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_load_data">
         <property name="text">
          <string>Загрузить данные в Систему</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_fit">
         <property name="text">
          <string>Обучить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>761</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Статус загрузки данных:</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>761</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_file_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>281</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="train_predictions_matrix">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>1291</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>991</width>
        <height>371</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>Тут будут данные по точностям обученной модели</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Работа с обученными моделями</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Подготовка данных (деление на тестовую и обучающие выборки)</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1342</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
